src/Evolution/Systems/Cce/Actions/InitializeFirstHypersurface.hpp

```diff
@@ -48,9 +48,8 @@ struct InitializeFirstHypersurface {
       const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
       const ArrayIndex& /*array_index*/, const ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) noexcept {
-    db::mutate_apply<typename Metavariables::cce_hypersurface_initialization>(
-        make_not_null(&box));
-    db::mutate_apply<InitializeGauge>(make_not_null(&box));
+    db::mutate_apply<InitializeJ::mutate_tags, InitializeJ::argument_tags>(
```

Comment:
FYI, if you renamed `mutate_tags` -> `return_tags` I think you could call this as `db::mutate_apply(db::get<Tags::InitializeJ>(box), make_not_null(&box))`

Reply:
unfortunately, no -- that's the first thing I tried, and that version only works if the `apply` is static, which it cannot be here due to the reliance on member variables.